sklearn/decomposition/_incremental_pca.py

```diff
@@ -289,8 +289,8 @@ def partial_fit(self, X, y=None, check_input=True):
             X -= col_batch_mean
             # Build matrix of combined previous basis and new data
             mean_correction = \
-                np.sqrt((self.n_samples_seen_ * n_samples) /
-                        n_total_samples) * (self.mean_ - col_batch_mean)
+                np.sqrt((n_total_samples) * (self.mean_ - col_batch_mean) /
+                self.n_samples_seen_ * n_samples)
```

Comment:
beware of the parenthesis. `(self.mean_ - col_batch_mean)` is outside of the `sqrt`.

```suggestion
                np.sqrt((self.n_samples_seen_ / n_total_samples) *
                        n_samples) * (self.mean_ - col_batch_mean)
```

Reply:
Thanks. That was causing some confusion on my end.